torch/onnx/symbolic_helper.py

```diff
@@ -163,33 +165,41 @@ def _parse_arg(
     )
 
 
+@_beartype.beartype
 def _node_get(node: _C.Node, key: str):
     """Gets attributes of a node which is polymorphic over return type."""
     assert isinstance(node, _C.Node)
     sel = node.kindOf(key)
     return getattr(node, sel)(key)
 
 
+@_beartype.beartype
 def _is_onnx_constant(value: _C.Value):
     """Whether a Value is an ONNX constant."""
     return value.node().kind() == "onnx::Constant"
 
 
-def _maybe_get_const(value: _C.Value, descriptor: _ValueDescriptor):
+@_beartype.beartype
+def _maybe_get_const(
+    value: Optional[Union[_C.Value, torch.Tensor, Number, Sequence]],
```

Comment:
Relaxed

Reply:
I wanted to (1)enable typing checking before (2) fixing the code. Modifying the annotations is a way to minimal change code to enable checking without modifying current behaviors. We can then optimize stuff from there